packages/ui/sass/utilities/text.scss

```diff
@@ -13,12 +13,47 @@
 .font-united {
   font-family: $united;
   text-transform: uppercase;
-  letter-spacing: normal;
+
+  &--h1 {
+    font-size: $h1-united-size;
+    line-height: 84px;
+    letter-spacing: 2px;
+  }
+
+  &--h2 {
+    font-size: $h2-united-size;
+    line-height: 60x;
+    letter-spacing: 2px;
+  }
+
+  &--h3 {
+    font-size: $h3-united-size;
+    line-height: 48px;
+    letter-spacing: 1px;
+  }
+
+  &--h4 {
+    font-size: $h4-united-size;
+    line-height: 40px;
+    letter-spacing: 0.5px;
+  }
+
+  &--h5 {
+    font-size: $h5-united-size;
+    line-height: 32px;
+    letter-spacing: 0.5px;
+  }
+
+  &--h6 {
+    font-size: $h6-united-size;
+    line-height: 20px;
+  }
 }
 
 .font-gotham {
-  font-family: $gotham;
+  font-family: 'Gotham';
```

Comment:
@marinagustint Why was this changed? By calling the Gotham font directly, you've prevented us from being able to easily change fonts in the future. You've also removed the fallback fonts. Any reason why we shouldn't use the $gotham variable here?